src/framework/execution/__tests__/logger.test.ts

```diff
@@ -294,7 +300,9 @@ describe('step event publishing', () => {
       events: [
         {
           name: 'step-failure',
-          description: 'Step "Mochi" failed to complete due to error.',
+          description: expect.stringContaining(
+            `Step "Mochi" failed to complete due to error. (errorCode=${error.code}`,
```

Comment:
Oh, just noticed there is no assertion that `errorId=` is included in the `description` text.

Reply:
I'll add that rnrq